versioned_sidebars/version-2.3.9-sidebars.json

```diff
@@ -0,0 +1,204 @@
+{
+  "docs": [
+    "about",
+    {
+      "type": "category",
+      "label": "Quick Start - V2",
+      "items": [
+        {
+          "type": "category",
+          "label": "Start With Locally",
+          "items": [
+            {
+              "type": "autogenerated",
+              "dirName": "start-v2/locally"
+            }
+          ]
+        },
+        {
+          "type": "category",
+          "label": "Start With Docker",
+          "items": [
+            {
+              "type": "autogenerated",
+              "dirName": "start-v2/docker"
+            }
+          ]
+        },
+        {
+          "type": "category",
+          "label": "Start With K8s",
+          "items": [
+            {
+              "type": "autogenerated",
+              "dirName": "start-v2/kubernetes"
```

Comment:
add this file path

https://github.com/apache/seatunnel/blob/2.3.9-release/docs/en/start-v2/kubernetes/helm.md